CODE_OF_CONDUCT.md

```diff
@@ -15,7 +15,7 @@ diverse, inclusive, and healthy community.
 ## Our Standards
 
 Examples of behavior that contributes to a positive environment for our
-community include:
+community includes:
```

Comment:
This is from upstream, please revert this change.

`include` is btw correct here as well.

Reply:
Unresolving, as this comment was marked resolved without an action that resolved the comment.